server/sandbox_run.go

```diff
@@ -405,8 +406,7 @@ func (s *Server) RunPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 
 	// setup the network
 	if !hostNetwork {
-		podNamespace := ""
-		if err = s.netPlugin.SetUpPod(netNsPath, podNamespace, id, containerName); err != nil {
+		if err = s.netPlugin.SetUpPod(netNsPath, namespace, name, id); err != nil {
```

Comment:
if I fix as per my comment above I can see the pod up and running with openshift-sdn address:
```
  9m    1m      35      kubelet, 192.168.122.186                                        WarningFailedSync      (events with common reason combined)
  10m   1m      45      kubelet, 192.168.122.186                                        NormalSandboxChanged   Pod sandbox changed, it will be killed and re-created.
  1m    1m      1       kubelet, 192.168.122.186        spec.containers{awesome}        NormalPulled           Container image "nginx:alpine" already present on machine
  1m    1m      1       kubelet, 192.168.122.186        spec.containers{awesome}        NormalCreated          Created container with id 62fa81061b36041ab4fb36f48903bc532c5ddbfd2a0a7a5f864c8f3628472432
  1m    1m      1       kubelet, 192.168.122.186        spec.containers{awesome}        NormalStarted          Started container with id 62fa81061b36041ab4fb36f48903bc532c5ddbfd2a0a7a5f864c8f3628472432

[root@fedora cri-o]# oc describe pod awesome | grep IP
IP:                     10.128.0.5
[root@fedora cri-o]# curl 10.128.0.5
<!DOCTYPE html>
<html>
<head>
<title>Welcome to nginx!</title>
<style>
    body {
        width: 35em;
        margin: 0 auto;
        font-family: Tahoma, Verdana, Arial, sans-serif;
    }
</style>
</head>
<body>
<h1>Welcome to nginx!</h1>
<p>If you see this page, the nginx web server is successfully installed and
working. Further configuration is required.</p>

<p>For online documentation and support please refer to
<a href="http://nginx.org/">nginx.org</a>.<br/>
Commercial support is available at
<a href="http://nginx.com/">nginx.com</a>.</p>

<p><em>Thank you for using nginx.</em></p>
</body>
</html>
```